sdks/java/io/elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.java

```diff
@@ -373,6 +376,8 @@ abstract static class Builder {
 
       abstract Builder setTrustSelfSignedCerts(boolean trustSelfSignedCerts);
 
+      abstract Builder setInvalidBulkEndpoint(boolean invalidBulkEndpoint);
```

Comment:
I don't particularly like the idea of having this builder method only for the sake of testing. I feel it will be very confusing to users to see this as an option when building configurations. Could you look for an alternative approach?

Reply:
I think this method and [this](https://github.com/apache/beam/pull/26424/files#diff-e119923661d41dfc1911da8f6e29cc0e2e9a0a50129718c1c20c7d1aa32119bfR658) are not visible from users.  Same approach as [this](https://github.com/apache/beam/blob/6ad2b12c0997ad70f54fe8367918f5a47d482b1b/sdks/java/io/elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.java#L1191-L1196).